src/utils.cpp

```diff
@@ -1628,6 +1629,179 @@ int get_skill(struct char_data *ch, int skill, int &target)
     }
   }
 
+  // Reflex Recorder, adds +1 die to Physical Skill tests for its associated skill. -Vile
+  // Utilizes Value 6 to reference its associated skill.
+  if (recorder)
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_EDGED_WEAPONS && GET_OBJ_VAL(bio, 5) == 1) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_POLE_ARMS && GET_OBJ_VAL(bio, 5) == 2) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_WHIPS_FLAILS && GET_OBJ_VAL(bio, 5) == 3) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_CLUBS && GET_OBJ_VAL(bio, 5) == 4) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_UNARMED_COMBAT && GET_OBJ_VAL(bio, 5) == 5) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
+    skill_dice += 1;
+  }
+  for (struct obj_data *bio = ch->bioware; bio; bio = bio->next_content)
+  if (skill == SKILL_CYBER_IMPLANTS && GET_OBJ_VAL(bio, 5) == 6) {
+    snprintf(ENDOF(gskbuf), sizeof(gskbuf) - strlen(gskbuf), "Reflex Recorder Skill Increase: +1. ");
```

Comment:
Since you're not changing any values here and are just appending text, it's better to use strlcat:
```c++
strlcat(gskbuf, "message", sizeof(gskbuf));
```
This avoids unexpected behavior from percent signs (if any are ever added), and also cuts down processing time by a truly miniscule amount. It's also a tiny bit more readable that way.